app/controllers/clients/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Clients::DashboardController < ApplicationController
+  def index 
+    @user = Account.where(user_id: current_user.id).first
```

Comment:
This is an account and not a `user`
```suggestion
    @account = Account.find_by(user_id: current_user.id)
```
but if `user has one account`

```suggestion
    @account= current_user.account
```